pkg/querier/blocks_store_queryable.go

```diff
@@ -595,8 +595,9 @@ func (q *blocksStoreQuerier) queryWithConsistencyCheck(
 	}
 
 	// We've not been able to query all expected blocks after all retries.
-	level.Warn(util_log.WithContext(ctx, spanLog)).Log("msg", "failed consistency check", "err", err)
```

Comment:
Note to reviewers: `err` was always `nil` here.

Reply:
oops, that was me, sorry